databaseUtils/setup-data/atttibute_table.csv

```diff
@@ -54,3 +54,41 @@ vegetation_height,cm,Average height of the vascular vegetation in the plot,measu
 moss_height,cm,Average height of the mosses,measured
 recorder,NA,Observer of the vegetation,defined
 date,yyyy-mm-dd,Date of observation,recorded
+solar_radiation,mw_hour_m-2,Solar radiation, measured
+total_N_red_oxi,mg_N_m2_yr-1,Total soil nitrogen reduced and oxidiced,measured
+geology,marble,Main bedrock types, defined
+land_use,free-range grazing cattle sheep reindeer,Main land-use type, defined
+value_numeric,Number,Numeric values,measured-recorded
+value_categorical,Text,Categorical values,defined-recorded
+norwegian_name,NA,Norwegian species name,recorded
+flowering_start,spring,Start of flowering defined as spring early mid or late summer,recorded
+flowering_finish,mid-summer,End of flowering defined as spring early mid late summer or autumn,recorded
+lower_vegetation_zone,Nem,Lower vegetation zone,recorded
+upper_vegetation_zone,Lalp,Upper vegetation zone,recorded
+habitat,Grassland,Habitat the species occurs,recorded
+soil_type,base-rich,Species association with base rich soils,recorded
+rarity,Common,Common or scattered species,recorded
+functional_group,forb,Plant functional group,recorded
+lifespan, annual,Species lifespan,recorded
+polyploid_2n,1-3,Species polyploidy level
+flowers_spring,1,Species with main flowering time in spring,recorded
+flowers_early_summer,1,Species with main flowering time in early summer,recorded
+flowers_mid_summer,1,Species with main flowering time in mid-summer,recorded
+flowers_late_summer,1,Species with main flowering time in late summer,recorded
+flowers_autumn,1,Species with main flowering time in autumn,recorded
+min_height,cm,Minimum species height,recorded
+max_height,cm,Maximum species height,recorded
+nemoral,0.5-1,Species main vegetation zone is nemoral,recorded
+boreal_nemoral,0.5-1,Species main vegetation zone is boreal to nemoral,recorded
+south_boreal,0.5-1,Species main vegetation zone is south boreal,recorded
+mid_boreal,0.5-1,Species main vegetation zone is mid boreal,recorded
+north_boreal,0.5-1,Species main vegetation zone is north boreal,recorded
+low_alpine,0.5-1,Species main vegetation zone is low alpine,recorded
+mid_alpine,0.5-1,Species main vegetation zone is mid-alpine,recorded
+high_alpine,0.5-1,Species main vegetation zone is high alpine,recorded
+grassland,1,Species main habitat type is grassland,recorded
+forest,1,Species main habitat type is forest,recorded
+rocky,1,Species main habitat type is rocky,recorded
+wetland,1,Species main habitat type is wetland,recorded
+lids_page,6,Page number of species in Lids Flora,recorded
+mossberg_page32,Page number of species in Mossbergs Flora,recorded
```

Comment:
missing comma on 94

Would prefer to keep **L**ids and **M**ossberg (no one is ever going to use these in code anyway)
Not sure recorded is correct as it implies we recorded these. Maybe "literature"